pages/sign-up.js

```diff
@@ -1,11 +1,32 @@
+import Link from "next/link";
 import { motion as m } from "framer-motion";
+import { AiOutlineTwitter } from "react-icons/ai";
 
-const singUp = () => {
+const signUp = () => {
   return (
-    <m.div className="signup" exit={{ opacity: 0 }}>
-      <h1>Sign up</h1>
+    <m.div
+      className="signup h-screen flex justify-center items-center"
+      exit={{ opacity: 0 }}>
+      <section className="flex-col flex gap-6">
+        <Link href="/host">
+          <button
+            type="button"
+            className="rounded-md px-8 py-3 flex gap-2 bg-[#1DA1F2]">
+            <AiOutlineTwitter className="h-6 w-6" />
+            <div>Sign-in as Host</div>
+          </button>
+        </Link>
+        <Link href="/host">
```

Comment:
Can we change the href to `speaker` when the user click the button?

Reply:
yup i can do that 👍🏻

Reply:
the speaker page is not available rn

Reply:
Okay i am changing it for the speakers section thanks for the review.